fdbserver/workloads/HealthMetricsApi.actor.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * WriteThrottling.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//
+// Created by sfc-gh-xwang on 6/30/20.
+//
+
+#include "fdbserver/TesterInterface.actor.h"
+#include "fdbserver/workloads/workloads.actor.h"
+#include "fdbserver/WorkerInterface.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+// workload description
+// This workload can be attached to other workload to collect health information about the FDB cluster.
+struct HealthMetricsApiWorkload : TestWorkload {
+    // Performance Metrics
```

Comment:
It seems that this file is formatted differently. Can you please run `clang-format` to make sure it's consistent with other stuff.